src/platforms/hosted/cmsis_dap.c

```diff
@@ -127,6 +127,22 @@ static size_t mbslen(const char *str)
 	return result;
 }
 
+// Return maximum length in bytes that can be sent in the 'data' payload of a
+// DAP transfer, given the interface type and (provided) DAP command header
+// size.
+static inline size_t dap_max_transfer_data(size_t command_header_len)
+{
+	size_t result = report_size - command_header_len;
+
+	// Allow for an additional byte of payload overhead
+	// when sending data in HID Report payloads
+	if (type == CMSIS_TYPE_HID) {
```

Comment:
The braces here are superfluous so please remove them. `result` could also be made const if the body of this statement was changed to `return result - 1U;`. Looks good otherwise

Reply:
Sure, done!